backend/test-data-source-badges.sh

```
@@ -0,0 +1,76 @@
+echo "2. API動作確認..."
+
+# APIレスポンスを取得してdataSourceフィールドを確認
+API_RESPONSE=$(curl -s "http://localhost:8081/api/v1/suggestions?situation=workplace&duration=5")
+
+if [ -z "$API_RESPONSE" ]; then
+    echo "❌ API応答なし"
+else
+    echo "✅ API応答あり"
+    
+    # dataSourceフィールドの存在を確認
+    if echo "$API_RESPONSE"  < /dev/null |  grep -q '"dataSource"'; then
```

Shellcheck (0.10.0)

[warning] Redirecting to 'echo', a command that doesn't read stdin. Did you want 'cat' instead?

(SC2217)